### ISSUE TITLE
Chop Tree

### ISSUE THREAD

**Issue by Shybella** [Shovinus/ChopDownUpdated#17]
I love this plugin but I also miss the tree just popping and dropping all the logs like most traditional tree cutting mods.

Is this possibly something that can be a configurable option?

Thanks.

**Comment by Shovinus** [comment]
Yes, download and veinminer tree feller log, there's hundreds of them, the point of the mod was to avoid unnatural vein mining but a) making it feel more like real life and b) making reaching the tops of tall trees not so annoying.
